website/index.html

```diff
@@ -108,8 +108,8 @@ <h2 id="example">Example <a href="#example">#</a></h2>
 const s = serve("0.0.0.0:8000");
```

Comment:
Thanks can you use `https://deno.land/x/std@0.2.11/http/server.ts` above so we don't hit this again?

Reply:
sure 👍